src/expressions.rst

```diff
@@ -4763,52 +4713,65 @@ A :t:`capture target` is either a :t:`variable` or a :t:`field` of a
 
 :dp:`fls_e70ywb8191h`
 The :t:`capturing environment` of a :t:`capturing expression` consists of all
-:t:`[capture target]s` that are defined outside the :t:`capturing expression`.
+:t:`captured` :t:`capture target` :t:`[value]s`.
```

Comment:
I would rephrase this as

The :t:`capturing environment` of a :t:`capturing expression` consists of the :t:`[value]s of all :t:`captured` :t:`[capture target]s`.